src/content-feature.js

```diff
@@ -116,13 +109,38 @@ export default class ContentFeature {
         if (this.#debugMessaging) return this.#debugMessaging
 
         if (this.platform?.name === 'extension' && typeof import.meta.injectName !== 'undefined') {
-            this.#debugMessaging = createMessaging({ name: 'debug', isDebug: this.isDebug }, import.meta.injectName)
+            const context = createMessagingContext({ name: this.name, isDebug: this.isDebug })
```

Comment:
Overall I think this should be created once in some global state but I don't really want to refactor this right now.